2024-06-14 10:12:11.380 [info] (Client) Pylance async client (2024.6.1) started with python extension (2024.8.0)
2024-06-14 10:12:12.417 [info] [Info  - 10:12:12 AM] (651) Pylance language server 2024.6.1 (pyright version 1.1.364, commit 0618acc5) starting
2024-06-14 10:12:12.418 [info] [Info  - 10:12:12 AM] (651) Server root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-06-14 10:12:12.441 [info] [Info  - 10:12:12 AM] (651) Starting service instance "ndao"
2024-06-14 10:12:12.490 [info] Auto-Indent enabled
2024-06-14 10:12:12.520 [info] Auto-Indent enabled
2024-06-14 10:12:12.523 [info] [Info  - 10:12:12 AM] (651) Setting pythonPath for service "ndao": "/bin/python3"
2024-06-14 10:12:12.523 [info] [Info  - 10:12:12 AM] (651) Setting environmentName for service "ndao": "3.8.10 (global)"
2024-06-14 10:12:12.565 [info] [Info  - 10:12:12 AM] (651) Assuming Python version 3.8.10.final.0
2024-06-14 10:12:12.866 [info] [Info  - 10:12:12 AM] (651) No source files found.
2024-06-14 10:12:12.876 [info] Auto-Indent enabled
2024-06-14 10:12:13.406 [info] [Info  - 10:12:13 AM] (651) Background analysis(1) root directory: file:///home/ndao/.vscode-server/extensions/ms-python.vscode-pylance-2024.6.1/dist
2024-06-14 10:12:13.415 [info] [Info  - 10:12:13 AM] (651) Background analysis(1) started
